packages/gateway/src/constants.ts

```diff
@@ -9,13 +9,16 @@
  * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
  * graphic logo are (registered/a) trademark(s) of Plan International.
  */
+export const REDIS_HOST = process.env.REDIS_HOST || 'localhost'
 export const HOST = process.env.HOST || '0.0.0.0'
 export const PORT = process.env.PORT || 7070
 export const FHIR_URL = process.env.FHIR_URL || 'http://localhost:5001/fhir'
+export const CERT_PRIVATE_KEY_PATH =
```

Comment:
This is probably not needed anymore, right?

Reply:
Removed.